src/tool_cfgable.c

```diff
@@ -45,6 +45,7 @@ void config_init(struct OperationConfig *config)
   config->http09_allowed = FALSE;
   config->ftp_skip_ip = TRUE;
   config->file_clobber_mode = CLOBBER_DEFAULT;
+  config->mptcp = FALSE;
```

Comment:
this should not be needed, because of the `memset(config, 0, sizeof(struct OperationConfig));`

Also, see my comment in `struct OperationConfig`